tikv-configuration-file.md

```diff
@@ -1303,14 +1303,11 @@ Configuration items related to `raftdb`
 
 Configuration items related to Raft Engine.
 
-> **Warning:**
->
-> Raft Engine is an experimental feature. It is not recommended to use it in the production environment.
-
 ### `enable`
 
 + Determines whether to use Raft Engine to store raft logs. When it is enabled, configurations of `raftdb` are ignored.
-+ Default value: `"false"`
++ If you need to downgrade to a version earlier than v5.4.0, please disable Raft Engine and restart TiKV first.
++ Default value: `"true"`
```

Comment:
```suggestion
+ Default value: `"true"`

> **Note:**
>
> The data format of Raft Engine in v5.4.0 is not compatible with previous versions. Therefore, before downgrading a TiDB cluster to versions earlier than v5.4.0, you need to first disable Raft Engine (by setting the value of `engine` to `disable`) and restart TiKV. Otherwise, you cannot start the downgraded cluster.
```

Reply:
@tabokie @TomShawn I have updated the description, PTAL, thanks!